components/submit-button.jsx

```diff
@@ -45,11 +24,7 @@ var DonateButton = React.createClass({
       <div className="row">
         <div className="full">
           <button onClick={this.onClick} type="submit" className="btn large-label-size" id="donate-btn">
-            <ButtonContent
-              amount={this.props.amount}
-              submitting={this.props.submitting}
-              currency={this.props.currency}
-            />
+            {this.renderButton()}
```

Comment:
We want to display "submitting..." with a spinner if submitting state is true, so again, I create a render function with my new attempt at accepting conditionals inside react markup.

I also boiled down this component to be generic. It submits to where ever, and displays a spinner or `this.props.children`